core/src/test/java/io/questdb/griffin/SqlParserTest.java

```diff
@@ -481,6 +596,21 @@ public void testAsOfJoinSubQuerySimpleAlias() throws Exception {
         );
     }
 
+    @Test
+    @Ignore
```

Comment:
remove?

Reply:
it will fail, correlated sub-query is not fully implemented yet

Reply:
expected "" looks unfinished

Reply:
I removed it